go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/bazelbuild/buildtools v0.0.0-20230510134650-37bd1811516d
 	github.com/bazelbuild/rules_go v0.39.1
 	github.com/bmatcuk/doublestar v1.2.2
-	github.com/emicklei/proto v1.9.0
+	github.com/emicklei/proto v1.13.0
```

Comment:
This version update picks up 2 changes helpful to this PR:

1. The introduction of `proto.NoopVisitor` which we can embed to simplify `proto.Visitor` implementations.
2. The fix of https://github.com/emicklei/proto/issues/140 which had complicated parsing enum options out of proto files.